app/jobs/signal_adapter/attach_contributors_avatar_job.rb

```diff
@@ -4,8 +4,11 @@ module SignalAdapter
   class AttachContributorsAvatarJob < ApplicationJob
     queue_as :attach_signal_avatar
 
-    def perform(contributor)
-      avatar = "/app/signal-cli-config/avatars/profile-#{contributor.signal_phone_number}"
+    def perform(contributor_id:)
+      contributor = Contributor.find_by(id: contributor_id)
+      return unless contributor
+
+      avatar = "/app/signal-cli-config/avatars/profile-#{contributor.signal_uuid}"
```

Comment:
I think so because we only ever call this job when we connect someone and now we only have the option of using a person's uuid. After we sunset the current instances, we can completely rip out the `signal_phone_number` as it won't be used any longer.